apps/bot/src/profile/imageBuilder.ts

```diff
@@ -0,0 +1,392 @@
+
+    this.combineTextElements("g[id='Marriage Status Text Top'] > text", {
+      sortTspanElements: true,
+    });
+    this.combineTextElements("g[id='Marriage Status Text Bottom'] > text");
+    // HACK)) Should this whitespace even be there?
+    // FIXME)) If possible, fix in Figma and remove this hack
+    this.#svg("g[id='Marriage Status Text Bottom'] > text > tspan:nth(1)").remove();
+
+    // Fix text alignment for 'Exp Value'
+    this.createTextBoundingBox(
+      "text[id='Exp Value']",
+      402.5, // From Figma: 'Exp Value' -> Position -> X
+      270, // From Figma: 'Exp Value' -> Layout -> Width
+    );
+
+    // Fix text alignment for 'Level Value'
+    this.createTextBoundingBox(
+      "text[id='Level Value']",
+      408, // From Figma: 'Level Value' -> Position -> X
+      259, // From Figma: 'Level Value' -> Layout -> Width
+    );
+  }
+
+  /**
+   * Combine text elements matching a selector into a single element
+   * with multiple <tspan> children.
+   *
+   * Move the `x` and `y` attributes from the first matched <text> element,
+   * but remove them from all <tspan> elements afterwards.
+   *
+   * Move values of `font-size` and `fill` to individual <tspan> elements
+   * and remove them from the <text> element.
+   *
+   * @param selector Selector for <text> elements which should be combined
+   */
+  private combineTextElements(
+    selector: string,
+    options?: { sortTspanElements?: boolean },
+  ) {
+    // First <text> element - all <tspan> elements will be moved here
+    const firstTextElement = this.#svg(selector).first();
+
+    // First <tspan> element - we keep its position
+    // FIXME)) This could be incorrect when <text> or <tspan> elements
+    //         aren't ordered correctly.
+    const firstTspan = firstTextElement.children("tspan").first();
+
+    // Copy the position from <tspan> to <text>
+    const x = firstTspan.attr("x");
+    const y = firstTspan.attr("y");
+    firstTextElement.attr("x", x).attr("y", y);
+
+    const allTspanElements = this.#svg(selector).children("tspan");
+
+    // Move `font-size` and `fill` attrs to individual <tspan> elements
+    for (const element of allTspanElements) {
+      const tspan = this.#svg(element);
+      const parent = tspan.parent();
+      tspan.attr("font-size", parent.attr("font-size"));
+      tspan.attr("fill", parent.attr("fill"));
+    }
+    this.#svg(selector).removeAttr("font-size").removeAttr("fill");
+
+    // Move all <tspan> elements to the first <text> element
+    allTspanElements.appendTo(firstTextElement);
+
+    // Sort <tspan> elements after consolidating them into a single <text> element
+    if (options?.sortTspanElements ?? false) {
+      const sortedTspanElements = firstTextElement
+        .children("tspan")
+        .get()
+        .sort((a, b) => {
+          const aX = this.#svg(a).attr("x");
+          const bX = this.#svg(b).attr("x");
+          if (!aX || !bX) return 0;
+          return +aX - +bX;
```

Comment:
`-` operator autoconverts strings to numbers

Reply:
Yes, but `The right-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.ts(2363)`

Reply:
then please 
```suggestion
          return Number(aX) - Number(bX);
```
it's easier to understand the meaning